src/test/java/com/EighthLight/app/TestGame.java

```diff
@@ -1,5 +1,6 @@
 package com.EighthLight.app;
 
+import com.EighthLight.app.Interfaces.IPlayer;
```

Comment:
Unused import?

Reply:
used to set the type when creating an ArrayList of Players eg. ArrayList(IPlayer) players = new ArrayList();